### DISCUSSION TITLE
How send message server to all client connected (10-200 client)

### DISCUSSION THREAD

**Discussion by systemsiteseason** [jchristn/SuperSimpleTcp#69] [General]
I want my server send to all client. Is there a way of doing this effectively?

**Comment by jchristn** [comment]
Hi @systemsiteseason please refer to: https://github.com/jchristn/SimpleTcp/discussions/69

**Comment by ruudscheenen** [comment]
Hey @systemsiteseason
Comment might be a bit late but I think @jchristn meant to tag another post, not your own.
The post he was referring to was #64 
The solution there works like a charm!
